res/css/photon/label.css

```diff
@@ -18,3 +18,7 @@
 .photon-label-micro {
   font-size: 11px;
 }
+
+.photon-label-horiz-padding {
```

Comment:
In our project, we have many horizontal form elements, so I thought it made sense to have it in the photon styles.